roles/infrastructure/krb5_client/tasks/freeipa.yml

```diff
@@ -23,12 +23,21 @@
     ipaclient_servers: "{{ groups['krb5_server'] }}"
   when: "krb5_kdc_type == 'Red Hat IPA' and 'krb5_server' in groups"
 
-- name: Set sssd to enumerate users and groups
```

Comment:
What happened to this SSSD bit? Not sure it should have been in freeipa.yml, but does it need to go somewhere else? Or have we just decided we don't want it?

Reply:
It was being used by Ranger to get the users and groups. It's a tricky decision as there are some environments where this will cause a lot of problems (envs with lots of users and groups). The playbook already configures all of the LDAP user-sync properties for Ranger but it won't enable the LDAP resolver automatically as the unix shell resolver is required for the first run. It's a manual task to switch that post-deployment. It is definitely a trade-off that needs to be considered.